RFC/src/Glossary.md

```diff
@@ -242,6 +249,18 @@ updating [digital asset]s living on the Tari network.
 
 A Tari Wallet is responsible for managing key pairs, and for constructing and negotiating [transaction]s for transferring and receiving [tari coin]s on the [Base Layer]. 
 
+## Pruning horizon
+
+[pruninghorizon]: #pruninghorizon	"Block height at which  pruning must commence"
```

Comment:
double space between "which" and "pruning"

Reply:
Is this statement a `must` statement? It seems a little aggressive to use must which is usually reserved for requirements and this horizon is a settable value. I would suggest must = will